packages/uniforms-material/src/wrapField.tsx

```diff
@@ -41,4 +37,22 @@ function wrap(
   );
 }
 
-export default Object.assign(wrap, { _filterDOMProps, _filterDOMPropsList });
+declare module 'uniforms' {
+  interface FilterDOMProps {
+    checkboxes: never;
+    disableItem: never;
+    fullWidth: never;
+    helperText: never;
+    margin: never;
+    variant: never;
+  }
+}
+
+filterDOMProps.register(
+  'checkboxes',
+  'disableItem',
+  'fullWidth',
+  'helperText',
+  'margin',
+  'variant',
+);
```

Comment:
I'd say it should be extracted to a separate PR.

Reply:
As discussed in a meeting, it's good that it's fixed but in the future let's split unrelated changes to other PRs.